packages/gatsby-plugin-utils/README.md

```diff
@@ -76,7 +76,46 @@ Here's a list of features:
 ```js
 const { hasFeature } = require(`gatsby-plugin-utils`)
 
-if (!hasFeature(`image-service`)) {
-  // You can polyfill image-service here so older versions have support as well
+if (!hasFeature(`image-cdn`)) {
+  // You can polyfill image-cdn here so older versions have support as well
+}
+```
+
+### Add ImageCDN support
+
+Our new ImageCDN allows source plugins to lazily download and process images. if you're a plugin author please use this polyfill to add support for all Gatsby V4 versions.
+
+For more information check (TODO)[]
+
+#### Example
+
+```js
+const {
+  addRemoteFilePolyfillInterface,
+  polyfillImageServiceDevRoutes,
+} = require(`gatsby-plugin-utils/pollyfill-remote-file`)
+
+/** @type {import('gatsby').createSchemaCustomization} */
```

Comment:
We normally don't use type hints in our docs, I'd skip them

Reply:
But they are soo good :joy: I'll remove it